cmd/restic/cmd_snapshots.go

```diff
@@ -231,20 +333,60 @@ type Snapshot struct {
 	ShortID string     `json:"short_id"`
 }
 
+// SnapshotGroup helps to print SnaphotGroups as JSON with their GroupReasons included.
+type SnapshotGroup struct {
+	GroupKey  groupKey
```

Comment:
Add so-called struct tags here to set the names in the JSON output
```suggestion
	GroupKey  groupKey `json:"group_key"`
```